subprojects/build-cache/src/main/java/org/gradle/caching/internal/CacheableEntity.java

```diff
@@ -16,9 +16,11 @@
 
 package org.gradle.caching.internal;
 
+import org.gradle.api.Describable;
+
 /**
  * An entity that can potentially be stored in the build cache.
  */
-public interface CacheableEntity {
+public interface CacheableEntity extends Describable {
     String getPath();
```

Comment:
Shall we call this `identity` now?

Reply:
Renamed, along with the property in the origin metadata file.